examples/watch/pod_namespace_watch.py

```diff
@@ -44,6 +44,7 @@ def main():
             w.stop()
```

Comment:
if 10 namespace events were printed and `w.stop()` got called. Does the example continue to print the pod events? Could you verify that?